packages/stark-ui/src/modules/input-mask-directives/directives/text-mask-base.directive.ts

```diff
@@ -0,0 +1,139 @@
+import { AnyMaskedOptions } from "imask";
+import { StarkNumberMaskConfig } from "./number-mask-config.intf";
+import { StarkTextMaskBaseConfig } from "./text-mask-config.intf";
+import { IMaskDirective, IMaskFactory } from "angular-imask";
+import {
+	AfterViewInit,
+	ElementRef,
+	Inject,
+	Injectable,
+	OnChanges,
+	OnDestroy,
+	Optional,
+	PLATFORM_ID,
+	Renderer2,
+	SimpleChange,
+	SimpleChanges
+} from "@angular/core";
+import { COMPOSITION_BUFFER_MODE } from "@angular/forms";
+import { StarkTimestampMaskConfig } from "./timestamp-mask-config.intf";
+
+export type MaskConfigType = StarkTextMaskBaseConfig | StarkNumberMaskConfig | StarkTimestampMaskConfig | string | boolean;
+
+@Injectable()
```

Comment:
Is the `@Injectable()` needed ? 
I don't see the `TextMaskBaseDirective` in the providers of `StarkInputMaskDirectivesModule` and I don't think it is necessary for this directive.
Could you remove it ? 😊

Reply:
packages/stark-ui/src/modules/input-mask-directives/directives/abstract-stark-text-mask-base-directive.service.ts:27:23 - error NG2007: Class is using Angular features but is not decorated. Please add an explicit Angular decorator.

27 export abstract class AbstractStarkTextMaskBaseDirective<
when I tried to build the app without @Injectable()